core/modules/filter/js/filter.js

```diff
@@ -333,6 +329,87 @@ Backdrop.behaviors.editorImageLibrary = {
         var $submit = $form.find('.form-actions input[type=submit]:first');
         $submit.trigger('mousedown').trigger('click').trigger('mouseup');
       });
+
+    // Lock image aspect ratio.
+    var naturalDimensions = {
+      width: null,
+      height: null
+    };
+    // When editing a previously added image or upload a new one.
+    var existingFile = $('.filter-format-editor-image-form .form-managed-file a').attr('href');
+    if (typeof existingFile !== 'undefined') {
+      let img = new Image();
```

Comment:
This seems a bit hacky, but I can't think of another way, other than perhaps attaching a listener to the actual img element in the document... but because this img is added by ckeditor, it may be difficult to do.